tests/integration/test_query.py

```diff
@@ -310,6 +347,27 @@ def test_filters(index, query):
     t = Text("job") % ""
     search(query, index, t, 7)
 
+    # Timestamps
+    t = Timestamp("last_updated") > sample_datetimes["mid"]
+    search(query, index, t, 2)
+
+    t = Timestamp("last_updated") >= sample_datetimes["mid"]
+    search(query, index, t, 5)
+
+    t = Timestamp("last_updated") < sample_datetimes["high"]
+    search(query, index, t, 5)
+
+    t = Timestamp("last_updated") <= sample_datetimes["mid"]
+    search(query, index, t, 5)
+
+    t = Timestamp("last_updated") == sample_datetimes["mid"]
+    search(query, index, t, 3)
+
+    t = Timestamp("last_updated").between(
```

Comment:
An `inclusive` kwarg matching the behavior in Pandas for `date_range` could work well.